pkg/evaluator/evaluator_test.go

```diff
@@ -133,6 +134,36 @@ print f2
 	assert.Equal(t, want, got)
 }
 
+func TestErrVarNotSet(t *testing.T) {
+	prog := `
+fn "hello"
+
+v := "world"
+
+func fn s:string
+    print s v  // line 7
+end
+`
+
+	rt := &testRT{}
+	rt.UnimplementedRuntime.print = rt.Print
+	eval := NewEvaluator(DefaultBuiltins(&testRT{}))
+	err := eval.Run(prog)
+	assert.Equal(t, true, errors.Is(err, ErrVarNotSet))
+	evalErr := &Error{}
+	assert.Equal(t, true, errors.As(err, &evalErr))
+	assert.Equal(t, "line 7 column 13", evalErr.token.Location())
+}
+
+func TestLenString(t *testing.T) {
+	prog := `
+print (len "🌟⭐️✨🌙🪐") // 6 runes, 6 graphemes
```

Comment:
This looks like 5 graphemes to me. perhaps remove the star since you do a test with that next?